packaging/db/zip/linux/ignite3db

```diff
@@ -59,7 +59,20 @@ start() {
 }
 
 stop() {
-  kill -9 "$(cat ${IGNITE_HOME}/pid)"
+  ignite_pid = "$(cat ${IGNITE_HOME}/pid)"
+  kill -15 "$ignite_pid"
+
+  max_attempts=10
+  count=0
+  while [ ps -p "$ignite_pid" > /dev/null ] && [ "$count" -lt "$max_attempts" ]; do
+    sleep 10
+    count=$((count + 1))
+  done
+
+  if [ ps -p "$ignite_pid" > /dev/null ]; then
```

Comment:
no need to use  [ ] at all.
Just 

if ps -p "$ignite_pid" > /dev/null; then

fi